test/vic-v2/00-all-fields.e2e.spec.js

```diff
@@ -45,21 +47,84 @@ const runTest = E2eHelpers.createE2eTest(
         .click('.form-panel .usa-button-primary');
       E2eHelpers.expectNavigateAwayFrom(client, '/contact-information');
 
+      // Photo page
+      client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(3)', 'progress-segment-complete');
+      client.waitForElementVisible('label.usa-button.usa-button-secondary', Timeouts.normal);
+      client.axeCheck('.main');
+
+      // Disable upload button style to reveal input for test
+      /* HACK: style overridden so browser driver can find/interact with hidden inputs
+         (see https://github.com/nightwatchjs/nightwatch/issues/505) */
+      client
+        .execute("document.getElementsByName('fileUpload')[0].style.display = 'block';");
+      // HACK: waitforElementVisible did not work but this does 
+      client.elementIdDisplayed('errorable-file-input-11');
+
+      // sighted path
+      // upload photo
+      client
+        .setValue('input[name="fileUpload"]', require('path').resolve(`${__dirname}/examplephoto.png`));
+
+      // crop photo
+      client.waitForElementVisible('.cropper-container-outer', Timeouts.normal);
+      client.click('.form-panel .usa-button-primary');
```

Comment:
Would visual regression tests be the best way to test the cropper's features?

Reply:
Could verify some of the button disabling behavior by rotating a wide image once (move up, move right, move left should all be disabled after this). That would validate that the canvas loaded the image and that the buttons can be disabled. 

Verifying the position of the image would require grabbing the image data from cropperjs.

Reply:
Yeah, it's going to be tough to test a lot of this. Our visual regression testing set up isn't really at the point where we could use it for this yet.

Reply:
Perhaps this testing should be done in tandem with the work to split the cropper component up? It sounds like we might not need to use nightwatch for some of these things. If so, I can create a ticket to describe this work.